File: message-api-schema-validator/src/main/java/uk/ac/jisc/rdss/tests/term/update/TermUpdateRequestTest.java
package uk.ac.jisc.rdss.tests.term.update;

import org.junit.Test;

import uk.ac.jisc.rdss.tests.AbstractSchemaValidatorTest;

public class TermUpdateRequestTest extends AbstractSchemaValidatorTest {

    @Override
    protected String getSchemaFileName() {
        return "messages/term/update/request_schema.json";
    }

    @Test
    public void validateRequest() throws Exception {
        validateJson("messages/term/update/request.json");
    }
}

File: message-api-schema-validator/tests/term/term_update_request_test.py
from tests.abstract_schema_validator_test import AbstractSchemaValidatorTest
from unittest import TestCase


class TermUpdateRequestTest(AbstractSchemaValidatorTest, TestCase):
    def get_json_schema_file_name(self):
        return "messages/term/update/request_schema.json"

    def test_validate_request(self):
        self.validate_json("messages/term/update/request.json")
